Sources/BlogModuleApi/Objects/BlogPost/BlogPostListObject.swift

```diff
@@ -10,15 +10,28 @@ import Foundation
 public struct BlogPostListObject: Codable {
 
     public var id: UUID
-    public var title: String
+    public var title: String?
```

Comment:
Same thing here with the title, why the ?

Reply:
For this, actually this is in the RestAPI return. The idea, is when it is deleted, you just return the timestamp, and ID, in ou case we don't return any more data, not even the title